tests/test_restart.py

```diff
@@ -0,0 +1,445 @@
+#!/usr/bin/env python
+''' VIC exact restart testing '''
+
+import numpy as np
+import datetime
+import os
+import glob
+import xarray as xr
+from test_utils import read_vic_ascii
+
+
+# -------------------------------------------------------------------- #
+def prepare_restart_run_periods(restart_dict, state_basedir, statesec):
```

Comment:
we just removed these comment lines from `run_tests.py` so probably best to do the same here.